cocos/3d/CCMesh.cpp

```diff
@@ -350,6 +350,7 @@ void Mesh::draw(Renderer* renderer, float globalZOrder, const Mat4& transform, u
 
 
     _meshCommand.setSkipBatching(isTransparent);
+    _meshCommand.setTransparent(isTransparent);
```

Comment:
If the `Transparent NavMesh` works wrongly, what about transparent `Sprite3D`, it also use `MeshCommand`. 
My proposal is that if the bug exists in `Sprite3D`, it is better to solve it in another pull request. Then if our user meet this bug, he can merge the pull request easily by solve the bug.
BTW, `MeshCommand` is used by many 3D features, but we changed it here, is there any side effects on other 3D features?